packages/sitecore-jss/src/site/graphql-siteinfo-service.ts

```diff
@@ -0,0 +1,132 @@
+import { GraphQLClient, GraphQLRequestClient } from '../graphql';
+import debug from 'debug';
+import { CacheClient, CacheOptions, MemoryCacheClient } from '../cache-client';
+
+const headlessSiteGroupingTemplate = 'E46F3AF2-39FA-4866-A157-7017C4B2A40C';
+
+const defaultQuery = /* GraphQL */ `
+  {
+    search(
+      where: { name: "_template", value: "${headlessSiteGroupingTemplate}", operator: EQ }
+    ) {
+      results {
+      ... on Site_e46f3af239fa4866a1577017c4b2a40c {
```

Comment:
Looks like we have the same problem here (auto-gen'd type which won't work elsewhere)

Reply:
The name will be consistent across instances, it's the name of template + template ID
We do need to ask DevEx to give it a user friendly name though.